src/pytest_inmanta_lsm/failed_resources_logs.py

```diff
@@ -34,9 +34,8 @@ def _extract_logs(self, get_version_result: Dict[str, Any]) -> List[Tuple[str, s
         for resource in get_version_result["resources"]:
             resource_id = resource["resource_id"]
 
-            # Only interested in failed resources
-            if resource["status"] != "failed":
-                continue
+            if resource["status"] in ["deploying", "skipped", "failed"]:
```

Comment:
This class is only instantiated at https://github.com/inmanta/pytest-inmanta-lsm/blob/afe9d375a9ae20105227440aee6375d7478eb82d/src/pytest_inmanta_lsm/managed_service_instance.py#L352. That method explicitly defines a collection of bad_states. Why don't we check `resource["status"]` against those bad states instead of comparing it against the above-mentioned hard coded values?

Reply:
The bad_states there are service states like "up" or "delete_validating_failed"

Reply:
I see.